test.js

```diff
@@ -76,6 +76,21 @@ resetMock({status: 0, stdout: 'null'})
 result = dockerLambda({returnSpawnResult: true})
 result.should.eql({status: 0, stdout: 'null'})
 
+// Should not fail if stdout contains logging
+resetMock({status: 0, stdout: 'Test\nResult\n{"success":true}'})
+result = dockerLambda()
+result.should.eql({success: true})
+
+// Should return last stdout entry if it's not JSON
```

Comment:
I'm not sure this is correct – the result should only ever come from the invocation of the Lambda function. I think it's probably best to just return `undefined` if the `JSON.parse` fails, because that should be an anomaly (like the function crashed or something).